connect/hwinfo.go

```diff
@@ -130,3 +130,26 @@ func hostname() string {
 	}
 	return ip
 }
+
+func readValues(arg string) (map[string]string, error) {
+	output, err := execute([]string{"read_values", arg}, false, nil)
```

Comment:
It would be good to add some info to the code and/or commit message about this being specific to S390.